proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.audit.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+// ActorType is the type of actor that caused the audited event.
+enum ActorType {
+  ACTOR_TYPE_UNSPECIFIED = 0;
+  ACTOR_TYPE_USER = 1;
```

Comment:
it looks like the `ACTOR_TYPE_` prefix [causes stutter downstream](https://github.com/bufbuild/buf/pull/1366/files#r959129560)

Reply:
Resolved by https://github.com/bufbuild/buf/pull/1366#discussion_r959546155